### PR TITLE
feat: added Issue component

### PR DESCRIPTION
**Pull request by xenock**
Making first version of Issue component due to replace _Task_ one in proper moment.

This PR includes:
- An issue can have 4 states: success, warning, info, error. *Default*:  error
- An issue can have a link. *Default*: null.
- If an Issue have a link, it will paint a right chevron.

### CODE REVIEW

File: src/Issue/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+const IssueIcon = ({ variant }) =>
+  ({
+    success: <Icon name="success" className="ola_issue-status" />,
+    warning: <Icon name="warning" className="ola_issue-status" />,
+    suggested: <Icon name="help" className="ola_issue-status" />,
+  }[variant] || <Icon name="error" className="ola_issue-status" />)
+
+const IssueTitle = ({ title, link }) => (
+  <>
+    <span className="ola_issue-title">{title}</span>
```
Comment (oscarotero) [inline src/Issue/index.js:14]: The proper tag for title should be `<strong>`

File: src/Issue/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    {link && <Icon name="chevronRight" />}
+  </>
+)
+
+const Issue = ({ title, variant, link }) => (
```
Comment (oscarotero) [inline src/Issue/index.js:19]: I miss a `size` property with two possible values: `small` and `medium`:
- `small`: The title is `callout` (and bold)
- `medium`: The title is `body` (and bold).

**Review by oscarotero — CHANGES_REQUESTED** [review]

File: src/Issue/style.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+.ola_issue-header {
+    font: var(--font-body);
+    color: var(--black);
+    list-style: none;
+    display: flex;
+    align-items: center;
+    padding: 0 var(--column-gap-3);
+    cursor: default;
```
[the remark this answers is not in the document]
Comment (diego-pascual) [reply src/Issue/style.css]: Done!

File: src/Issue/index.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  <a href={link} className={`ola_issue is-${variant}`}>
```
[the remark this answers is not in the document]
Comment (diego-pascual) [reply src/Issue/index.js]: Done!

File: src/Issue/index.js (hunk 2 above)
Comment (diego-pascual) [reply src/Issue/index.js]: done!

**Pull request merged**